### DISCUSSION TITLE
Clear filters from viewmodel?

### DISCUSSION THREAD

**Discussion by LoyalServant** [dotnet/DataGridExtensions#98] [Q&A]
If I have a datagrid with an ItemsSource bound to an ObservableCollection, and that ObservableCollection changes, how do I clear the existing filters?
In my case I am clearing the collection, and adding new items so the existing filter is in place and working, but it no longer applies. This means the user has to manually remove it - if they realise that's why they are not seeing what they expect to see.

Thoughts?
Thanks all.

**Comment by tom-englert** [comment]
Hard to say without seeing the sources.

Can you add your use case to the sample app and provide a PR?
